Cargo.toml

```diff
@@ -56,7 +61,7 @@ serde_json = "1.0.104"
 sqlx = { version = "0.7", default-features = false }
 thiserror = "1.0"
 tokio = { version = "1.30.0", default-features = false }
-wkt = "0.10.3"
+wkt = { git = "https://github.com/georust/wkt", rev = "a26848c97686245548a43375133f41e77322247d" }
```

Comment:
you can just add this to the `patch.crates-io` section below. Not a biggie for a WIP of course